src/pages/general/login/components/loginForm/loginForm.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import {LoginCredentials} from '../../../../../model/login/loginCredentials';
+import {ILoginErrors} from '../../../../../model/login/loginErrors';
+import {HeaderComponent} from './components/header';
+import {FormComponent} from './components/form';
+
+interface IProps {
+  loginCredentials: LoginCredentials;
+  loginErrors: ILoginErrors;
+  updateLoginInfo: (viewModel: LoginCredentials, fieldName: string, value: string) => void;
+  loginRequest: (loginCredentials: LoginCredentials) => void;
+}
+
+export const LoginFormComponent = (props: IProps) => {
+  return (
+    <div className="container">
+      <div className="row">
+        <div className="col-md-4 col-md-offset-4">
```

Comment:
I think @rumbado already defined a centeredContainer component, not sure if it's in master but it would wrap  "ro" and "md4" stuff

This code 

```
<div className ="row">
   <div className="cold-md-4 col-mf-offset-4>
      ...
   </div>
</div>
```

Turns into this code
```html
<CenteredContainer>
...
</CenteredContainer>
```